<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelCustomer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Customers</string>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="customerTableWidget">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelAccount">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Accounts</string>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="accountDetailTable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addCustomerButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>Add a Customer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteCustomerButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>Delete a Customer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
